packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -248,13 +248,13 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection impleme
     /**
      * Error raising for socket.io issues
      */
-    protected createErrorObject(handler: string, error?: any, canRetry = true): DriverError {
+    protected createErrorObject(handler: string, error?: unknown, canRetry = true): DriverError {
         // Note: we suspect the incoming error object is either:
         // - a string: log it in the message (if not a string, it may contain PII but will print as [object Object])
         // - a socketError: add it to the OdspError object for driver to be able to parse it and reason
         //   over it.
-        if (canRetry && typeof error === "object" && error !== null) {
```

Comment:
This looks right on surface, but this check was here likely for a reason. Can you please dig through history and see why it was here? I worry that this is going to introduce some kind or regression